floaters/src/main/java/com/mdg/droiders/floaters/ExpandedWindow.java

```diff
@@ -92,23 +93,31 @@ public boolean onTouch(View v, MotionEvent event) {
                                         event.getRawX() - container.getInitialTouchX());
                                 y = (int) (container.getInitialY() +
                                         event.getRawY() - container.getInitialTouchY());
+                                // This piece of code is to keep the floating view in screen bounds
+                                // if the view moves out of screen, it becomes invisible for some reason
                                 if (x <= window.getWidth() - mfloatingView.getWidth())
                                     lp.leftMargin = x;
                                 else
                                     lp.leftMargin = window.getWidth() - mfloatingView.getWidth();
 
                                 if (y <= window.getHeight() - mfloatingView.getHeight())
                                     lp.topMargin = y;
-                                else lp.topMargin = window.getHeight() - mfloatingView.getHeight();
+                                else
+                                    lp.topMargin = window.getHeight() - mfloatingView.getHeight();
                                 window.updateViewLayout(mfloatingView, lp);
                                 return true;
                             }
                             case MotionEvent.ACTION_UP: {
                                 // Since we have implemented the onTouchListener therefore we cannot implement onClickListener
                                 // Therefor we make changes to the onTouchListener to handle touch events
+                                RelativeLayout.LayoutParams lp = containerFloat.getRelativeParams();
```

Comment:
\[Checkstyle\] WARNING: 'member def type' have incorrect indentation level 32, expected level should be one of the following: 24, 26, 28\.